src/main/java/uk/gov/pay/webhooks/deliveryqueue/managed/SendAttempter.java

```diff
@@ -53,13 +61,13 @@ public SendAttempter(WebhookDeliveryQueueDao webhookDeliveryQueueDao,
     }
 
     public void attemptSend(WebhookDeliveryQueueEntity queueItem) {
+        var webhook = queueItem.getWebhookMessageEntity().getWebhookEntity();
         var retryCount = webhookDeliveryQueueDao.countFailed(queueItem.getWebhookMessageEntity());
         Instant start = instantSource.instant();
 
         try {
-            var url = queueItem.getWebhookMessageEntity().getWebhookEntity().getCallbackUrl();
-            var uri = new URI(url);
-            
+            var uri = new URI(webhook.getCallbackUrl());
```

Comment:
We’re using `URI` rather than `URL` here, the latter being what we use in the validation. We should probably standardise on `URL`.